sample-configuration.yaml

```diff
@@ -0,0 +1,3 @@
+dcos_generate_config_url: null
```

Comment:
FWIW a follow-up will get rid of this and make `generate_config_path` required. Tests should not rely on the network where possible. Also, the benefit of just putting a URL here is outweighed by the pain on the second test run. Also the confusion of "what happens when I provide a URL and a path?" is not worth it.